### ISSUE TITLE
Left column gutter not displaying

### ISSUE THREAD

**Issue by cold-brewed** [neovide/neovide#1473]
**Describe the bug**
When using the `vim-gitgutter` plugin the column on the left side that displays changes does not appear. The column **does** appear properly when using `nvim` in the terminal.

I am able to reproduce this with a bare `nvim` config (only the minimal amount needed to use `Plug`) and only the `vim-gitgutter` plugin enabled. The left column / gutter still does not display in `neovide` but does display in `nvim`.

```vimrc
" init.vim used for test
set runtimepath^=~/.vim runtimepath+=~/.vim/after
let &packpath = &runtimepath
call plug#begin()
Plug 'airblade/vim-gitgutter'
call plug#end()
```

Note: All of my other vim plugins (when enabled) seem to work without issue. I can also `set number` and the number column does appear. It doesn't seem to make a different on the git gutter column appearing, however.

**To Reproduce**
1. Install `vim-gitgutter` plugin
2. Make changes to a tracked file
3. Observe no gutter
4. Alternately, you can simply open a tracked file and normally the gutter would still be visible even without changes. However, it doesn't display in `neovide`

**Expected behavior**
The gutter column should display on the left side.

**Screenshots**

An example of the gutter appearing in `nvim`:
![nvim-with-gutter](https://user-images.githubusercontent.com/47337035/184169375-815149a7-b68a-4a82-a415-cf93789d2040.png)

An example of the issue in `neovide`. **Do note** that the white area to the left is the window beneath the `neovide` window and not `neovide` itself. This is intentionally a high-contrast white in order to clearly see where the left side of the `neovide` window ends:
![neovide-no-gutter](https://user-images.githubusercontent.com/47337035/184171641-290f789a-f729-4bde-bb90-714cc757e9aa.png)

**Desktop (please complete the following information):**
 - OS: macOS 12.5
 - Neovide Version 0.9
 - Neovim Version v0.7.2

**Additional context**
I've been able to reproduce this on two different macOS hosts.

**Comment by MultisampledNight** [comment]
Thanks for reporting! Though I can't seem to reproduce the issue, with this `init.vim` and a `:PlugInstall`:

```vim
call plug#begin()
Plug 'airblade/vim-gitgutter'
call plug#end()
```

Starting neovim through `nvim -u ./init.vim` and Neovide through `neovide -- -u ./init.vim`, I can see the statusbar in both neovim and Neovide:

![Neovim displaying a file with dummy text, with a signcolmun](https://user-images.githubusercontent.com/80128916/184301761-5acce8db-da8c-4a53-8b23-45961dd58997.png)
![Neovide showing the same behavior as above with neovim](https://user-images.githubusercontent.com/80128916/184301767-2713678d-699c-484c-b160-582fd95d31b5.png)

Which leaves us with the following possibilities:

- This is a platform-specific issue only reproducable on macOS. I'm using Arch Linux, for that matter.
- These two lines

  ```vim
  set runtimepath^=~/.vim runtimepath+=~/.vim/after
  let &packpath = &runtimepath
  ```

  add some variance in your local setup.

**Comment by cold-brewed** [comment]
Thank you for the response! After reading what you wrote, I was able to (I think) locate the issue. Perhaps it was a case of [rubber duck debugging](https://en.wikipedia.org/wiki/Rubber_duck_debugging) for me and I needed to talk it out 😄 

I had not thought about trying in linux. I went to install `neovide` in my `aarch64` linux vm. That was quite an experience thanks to `rust-skia` being a bit challenging to build on `aarch64`. I'll need to file an issue at their repo, but I was able to get through it. The `neovide` snaps don't have `aarch64` either.

I had the same results as you: it seemed to work fine on linux. However, this time around I installed `Plug` itself in the `nvim` directory instead of sourcing it through `vim`. The reason I had those extra lines like `runtimepath` in in my `init.vim` was because my `Plug` lines wouldn't work without it (and of course for sourcing my `.vimrc`). I had not thought to use `Plug` directly in my `nvim` install instead of using the regular `vim` sourcing because it otherwise seemed to work fine for all of the other plugins.

I went back to my mac and did the same thing: installed `Plug` directly in `nvim` as well as vim. And... now `vim-gitgutter` seems to work fine on `neovide`! Hooray! 

There may be a problem somewhere, but to be honest, I'm not sure if it's in `nvim`, `neovide`, `vim-gitgutter`, `Plug`, etc. But I consider this fixed. Thanks for the help. I'm very happy I can get to using `neovide` now! Great project!
